data.sql

```diff
@@ -1 +1,8 @@
-insert into animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, true, 10.23), ('Gambumon', '2018-11-15', 2, true, 8),('Pikachu', '2021-01-07', 1, false, 15.04), ('Devimon', '2017-3-12', 5, true, 11);
\ No newline at end of file
+insert into animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES 
+('Agumon', '2020-02-03', 0, true, 10.23), ('Gambumon', '2018-11-15', 2, true, 8),('Pikachu', '2021-01-07', 1, false, 15.04), ('Devimon', '2017-3-12', 5, true, 11);
+
+INSERT INTO animals (name,date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+ ('Charmander', '2022-2-8', 0, false, 11), ('Plantmon', '2021-11-15', 2, true, 5.7),
+ ('Squirtle', '1993-4-2', 3, false, 12.13), ('Angemon', '2005-6-12', 1,true, 45),
+  ('Boarmon', '2005-6-7',7,true, 20.4), ('Blossom', '1998-10-13', 3, true, 17), 
+  ('Ditto', '2022-5-14', 4, true, 22);
```

Comment:
- [ ] Good job adding data provided in the project requirements 👍 , however , I will suggest you to pay a closer attention to the fact that certains weight value has negative value , kindly check the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) and correct that 😄